Task7FinalReact/valery-youtube-searcher/src/App.js

```diff
@@ -0,0 +1,53 @@
+import cl from './App.module.css';
+import {useState} from "react";
+import {useDispatch, useSelector} from "react-redux";
+import SearchInput from "./components/SearchInput/SearchInput";
+import {setVideosFromYoutube, addVideosFromYoutube} from "./actions/videos";
+import Error from "./components/Error/Error";
+import VideoContainer from "./components/VideoContainer/VideoContainer";
+
+const App = () => {
+    const dispatch = useDispatch()
+    const videoItems = useSelector(state => state.videos.items)
+    const isFetching = useSelector(state => state.videos.isFetching)
+    const isFetchError = useSelector(state => state.videos.isFetchError)
+    const nextPageToken = useSelector(state => state.videos.nextPageToken)
+    const [searchVideoString, setSearchVideoString] = useState('')
+
+
+    function searchHandler(searchValue) {
+        if (searchValue === "") {
+            return
+        }
+        dispatch(setVideosFromYoutube(searchValue, 10))
+        setSearchVideoString(searchValue)
+    }
+
+    function endHandler() {
+        if (searchVideoString === "") {
```

Comment:
Please, add `eslint` and configure it using `airbnb-react` https://github.com/airbnb/javascript/tree/master/react

It's better to use `;` on the line endings.